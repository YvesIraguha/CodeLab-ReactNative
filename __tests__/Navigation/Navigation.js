import React from 'react';
import { shallow } from 'enzyme';

import Navigator from '../../src/navigation/Navigation';

describe('Navigation component', () => {
  test('should render the app container with routes', () => {
    const wrapper = shallow(<Navigator />);
    expect(wrapper).toMatchSnapshot();
  });
});
