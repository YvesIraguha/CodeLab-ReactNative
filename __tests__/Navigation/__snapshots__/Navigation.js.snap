// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation component should render the app container with routes 1`] = `ShallowWrapper {}`;
